projects/samples/devices/controllers/imu/imu.c

```diff
@@ -0,0 +1,215 @@
+/*
+ * Copyright 1996-2022 Cyberbotics Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/*
+   Description:   Example world for demonstrating the difference between InertialUnit measurements and the fusion of multiple
+   sensors of the MPU-9250 IMU.
+*/
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <time.h>
+#include <webots/accelerometer.h>
+#include <webots/compass.h>
+#include <webots/gyro.h>
+#include <webots/inertial_unit.h>
+#include <webots/motor.h>
+#include <webots/robot.h>
+
+#define RAD_TO_DEG 57.2958
+#define NB_STEPS 10
```

Comment:
10 iterations are not enough to observe a significant difference on the gyro result.
```suggestion
#define NB_STEPS 100
```